lib/internal/Magento/Framework/Mail/Message.php

```diff
@@ -25,35 +36,40 @@ public function __construct($charset = 'utf-8')
     protected $messageType = self::TYPE_TEXT;
 
     /**
-     * Set message body
+     * Set message type
      *
-     * @param string $body
+     * @param string $type
      * @return $this
      */
-    public function setBody($body)
+    public function setMessageType($type)
     {
-        return $this->messageType == self::TYPE_TEXT ? $this->setBodyText($body) : $this->setBodyHtml($body);
+        $this->messageType = $type;
+        return $this;
     }
 
     /**
-     * Set message body
-     *
-     * @return string
+     * @param null|object|string|\Zend\Mime\Message $body
+     * @return \Zend\Mail\Message
      */
-    public function getBody()
+    public function setBody($body)
     {
-        return $this->messageType == self::TYPE_TEXT ? $this->getBodyText() : $this->getBodyHtml();
```

Comment:
Fortunately `getBodyHtml` is only used in integration tests. I'm trying to fix them